Utilities/bootstrap

```diff
@@ -1131,7 +1133,10 @@ def main():
     if args.generate_xcodeproj:
         # We will use the `swift-package` binary we just built to generate the
         # Xcode project.
-        cmd = [swift_package_path, "generate-xcodeproj"]
+        cmd = [swift_package_path]
+        if args.enable_perf_tests:
+            cmd.extend(["-Xswiftc", "-DENABLE_PERF_TESTS"])
+        cmd.extend(["generate-xcodeproj"])
```

Comment:
Not important, but shouldn't this be `generate-xcodeproj -Xswiftc -DENABLE_PERF_TEST`?